src/main/java/frc/robot/commands/vision/VisionAim.java

```diff
@@ -29,7 +29,7 @@ public VisionAim(Robot robot)
     this.robot = robot;
 
     // subsystems that this command requires
-    addRequirements(robot.vision, robot.drivetrain, robot.shooter);
+    addRequirements(robot.drivetrain, robot.shooter);
```

Comment:
Won't this command technically write to the ring light?  I guess since it only happens once at the start and once at the end it shouldn't be a big deal.  I don't think any other command uses the ring light anyway.

Reply:
The auto aim and the auto speed have to both turn on and off the ringlight, and they both have to run at the same time, so they can't require vision. I have an idea for how to fix this.